README.md

```
@@ -1 +1,166 @@
+```
+
+### Requirements
+
+- LaTeX distribution (TeX Live, MiKTeX, etc.)
+- moderncv package
+- fontawesome5 package
+- ChkTeX (for linting)
+
+## CV Structure
+
+### Main CV Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)